storage/database/schema.go

```diff
@@ -94,16 +97,16 @@ type TxLookupEntry struct {
 	Index      uint64
 }
 
-// encodeBlockNumber encodes a block number as big endian uint64
-func encodeBlockNumber(number uint64) []byte {
+// encodeUint64 encodes a block number as big endian uint64
```

Comment:
For dbEntryType, did you change this name?
uint64 is necessary for dbEntryType?

Reply:
Same answer from line#192